test/integration/coders.test.ts

```diff
@@ -23,4 +27,15 @@ describe('LeafValueCoder', () => {
       expect(LeafValueCoder.decode(leaf, FIXED_NUMBER_PREFIX)).to.eql(result);
     });
   });
+
+  const bigints: (number | bigint)[] = [minInt224, -123456789, -33, -1n, 0n, 1n, 11, 123, maxInt224];
```

Comment:
we also need to test agains floating points numbers

Reply:
will push a commit with support for floating points

Reply:
added support and tests for floating point